BackEnd/src/main/java/com/unosquare/admin_core/back_end/controller/DashboardController.java

```diff
@@ -98,6 +98,7 @@ private void mapEmployeeSnapshotViewModel(EmployeeSnapshotDto m, EmployeeSnapsho
         employee.setName(m.getName());
         employee.setEmployeeId(m.getEmployeeId());
         employee.setEmail(m.getEmail());
+        employee.setClientName(m.getClientName());
```

Comment:
Can this mapper be completed with modelmapper? Just so we can keep our mapping logic separated